manticore/core/manticore.py

```diff
@@ -120,6 +130,7 @@ def raise_signal():
         self._terminated_states = self._manager.list()
         self._busy_states = self._manager.list()
         self._killed_states = self._manager.list()
+        self._log_queue = self._manager.Queue(15000)
```

Comment:
For background, see: [this comment](https://github.com/trailofbits/manticore/pull/1620/files#diff-326c1f66ee05ca61fe4dbebf5824c56eb6a2066374237b31f0be27712db8fe21R295-R300). They all used a queue in a previous iteration, which was changed in [this commit](https://github.com/trailofbits/manticore/pull/1620/commits/92c16300d1e0b6d569ddfef312b754c99188755d)

Because the queue is so much slower when it's full, I set the max size to triple that of the deques. That will increase memory usage but give us a little more leeway before becoming slow.

Reply:
Added an inline comment